packages/MainUI/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```

Comment:
Are these comments necessary?

Reply:
Yesss, these are the references directives of next, eliminate this references could provoke type errors with the next types, errors with imported images, etc, and is automatically generated by next when you run a dev env, you can ignore it for git, but is commonly accepted to include it